prototypes/python3/server/data_sources/mqtt/mqtt_data_source.py

```diff
@@ -275,21 +275,26 @@ def get_instance_by_id(self, element_id: str) -> Optional[Dict[str, Any]]:
             self.logger.info(f"Returning instance: {instance}")
             return instance
     
-    def get_relationship_types(self) -> List[str]:
-        return ["HasChildren", "HasParent"]
+    def get_instance_values_by_id(
+        self,
+        element_id: str,
+        startTime: Optional[str] = None,
+        endTime: Optional[str] = None,
+    ) -> Optional[Dict[str, Any]]:
+        value = self.get_topic_value(element_id)
+        return value
+
+    # No custom types
+    def get_relationship_types(self, namespace_uri: Optional[str] = None) -> List[Dict[str, Any]]:
+        return []
 
-    # TODO - need to fix this to return proper types
     def get_relationship_type_by_id(self, element_id: str) -> Optional[Dict[str, Any]]:
-        if element_id.lower() == "haschildren":
```

Comment:
I guess this is the work-around for the relationshiptypes "hack" imposed by the mock data? We should discuss how to fix this with the group. Will take it for now.